src/main/java/com/jagrosh/jmusicbot/commands/dj/ForceRemoveCmd.java

```diff
@@ -0,0 +1,60 @@
+package com.jagrosh.jmusicbot.commands.dj;
+
+import com.jagrosh.jdautilities.command.CommandEvent;
+import com.jagrosh.jmusicbot.Bot;
+import com.jagrosh.jmusicbot.audio.AudioHandler;
+import com.jagrosh.jmusicbot.commands.MusicCommand;
+
+
+public class ForceRemoveCmd extends MusicCommand {
+    public ForceRemoveCmd(Bot bot) {
+        super(bot);
+        this.name = "forceremove";
+        this.help = "removes all entries by the mentioned user from the queue";
+        this.arguments = "<@user|id>";
+        this.aliases = new String[]{"forcedelete"};
```

Comment:
In a similar vein to the [forceskip command](https://github.com/jagrosh/MusicBot/blob/master/src/main/java/com/jagrosh/jmusicbot/commands/dj/ForceskipCmd.java#L35), `modremove` and `moddelete` should also be aliases.